### PR TITLE
adds round six us epic copy & ticker test

### PR DESCRIPTION
**Pull request by tsop14**
## What does this change?
Updates US epic end of year test to round six. New control and four new variants. 
Adds a class to give padding to the quote. 
Updates template so that the highlighted (yellow) final sentence is part of the final paragraph (previously this was a new paragraph). @ionamckendrick 

## Screenshots
| control  | US_EoY_R7_OutsidePT | US_EoY_R7_Hope | US_EoY_R7_Rebecca | US_EoY_R7_John |
| ------- | ----------------------- |----------------- |--------------------- |------------- |
| screen shot 2018-12-12 at 14 36 49 |  screen shot 2018-12-12 at 14 35 19 | screen shot 2018-12-12 at 14 35 38  |  screen shot 2018-12-12 at 14 37 05 |  screen shot 2018-12-12 at 14 36 37 | 

## What is the value of this and can you measure success?
Driving traffic towards support site and contributions.

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-us-top-ticker.js [hunk 1]
```diff
@@ -33,32 +33,121 @@ const shorterCopy: AcquisitionsEpicTemplateCopy = {
...
+        'We’ve seen direct attacks on journalists in the past year, from the murder of Jamal Khashoggi to Trump’s incessant attacks on the media as &ldquo;the enemy of the people,&rdquo; and for a couple of decades we’ve seen the indirect attacks that are Silicon Valley’s siphoning off of advertising revenue and amplification of untruths for profit.',
+        'It costs a lot to send someone out to cover a campaign or to investigate a crime; it’s hard work that requires expertise and support from our readers. This year, The Guardian has covered everything from tech to feminism to Trump to fossil fuel politics. It is our editorial independence that has allowed us to deliver this fearless reporting; an independence that’s sometimes hard to find in other US-based media. We hope you appreciate our efforts.',
+        'The Guardian is asking our US readers to help us raise $1 million dollars by the new year to report on the most important stories in 2019. Every contribution, big or small, will help us hit our goal.',
+    ],
+    highlightedText:
+        'I hope you’ll consider making a year-end gift today to show your ongoing support for our independent journalism.Thank you.',
```
Comment (joelochlann) [inline static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-us-top-ticker.js]: space before `Thank you`

File: static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-us-top-ticker.js [hunk 2]
```diff
@@ -33,32 +33,121 @@ const shorterCopy: AcquisitionsEpicTemplateCopy = {
...
+        'Almost 100 years ago, in 1921, the editor of The Guardian argued that the principal role of a newspaper was accurate reporting, insisting that &ldquo;facts are sacred.&rdquo; We still hold that to be true. The need for a robust, independent press has never been greater, but the challenge is more intense than ever as digital disruption threatens traditional media’s business model. We pride ourselves on not having a paywall because we believe truth should not come at a price for anyone. Our journalism remains open and accessible to everyone and with your help we can keep it that way.',
+        'The Guardian is asking our US readers to help us raise $1 million dollars by the new year to report on the most important stories in 2019. Every contribution, big or small, will help us hit our goal',
+    ],
+    highlightedText:
```
Comment (joelochlann) [inline static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-us-top-ticker.js]: space before `Thank You`

File: static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-us-top-ticker.js [hunk 3]
```diff
@@ -33,32 +33,121 @@ const shorterCopy: AcquisitionsEpicTemplateCopy = {
...
+        'Almost 100 years ago, in 1921, the editor of The Guardian argued that the principal role of a newspaper was accurate reporting, insisting that &ldquo;facts are sacred.&rdquo; We still hold that to be true. The need for a robust, independent press has never been greater, but the challenge is more intense than ever as digital disruption threatens traditional media’s business model. We pride ourselves on not having a paywall because we believe truth should not come at a price for anyone. Our journalism remains open and accessible to everyone and with your help we can keep it that way.',
+        'The Guardian is asking our US readers to help us raise $1 million dollars by the new year to report on the most important stories in 2019. Every contribution, big or small, will help us hit our goal',
```
Comment (joelochlann) [inline static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-us-top-ticker.js]: full stop after `goal`

File: static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-us-top-ticker.js (hunk 1 above)
Comment (tsop14) [reply static/src/javascripts/projects/common/modules/experiments/tests/acquisitions-epic-us-top-ticker.js]: 🦅👀Thanks have updated all.

**Review by joelochlann — APPROVED** [review]
A few typos but other than that looks good!

**Pull request merged**